For Advanced/020.md

```diff
@@ -0,0 +1,42 @@
+# Elementary ID
+
+Many of the `selenium.` commands allow to select an element on a page in order to click it, get or set its attribute, or perform other actions. They use the `search` and the `by` arguments to pinpoint the required element: you provide the phrase to search for an element after the `search` argument and the element selector after the `by` argument.
+
+Open [Google](https://google.com) in your browser. Now, right-click the **Sign In** button in the top-right corner of the page and select `Inspect` (Chrome) or `Inspect Element` (Firefox, IE) from the context menu. A new pane will be displayed on the right side (Chrome) or at the bottom (Firefox, IE) of the browser, revealing the code behind the page. Take a look at it — specifically, at the selected line of the code:
```

Comment:
the developer tools pane will be displayed where a user previously placed it :p so it's not entirely true

Reply:
yep, it's all "by default". Do you think it's important to mention this? I guess, if a user placed it somewhere else, he/she already knows what inspection function is 😄

Reply:
I would add "by default" as you says

Reply:
I would add "by default" as you say

Reply:
```suggestion
Open [Google](https://google.com) in your browser. Now, right-click the **Sign In** button in the top-right corner of the page and select `Inspect` (Chrome) or `Inspect Element` (Firefox, IE) from the context menu. A new pane will be displayed on the right side (Chrome) or at the bottom (Firefox, IE) of the browser by default, revealing the code behind the page. Take a look at it — specifically, at the selected line of the code:
```